### PR TITLE
html file for blurjs

### PR DESCRIPTION
**Pull request by ieliz**

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+<meta charset="utf-8">
+<title>Hello OpenCV.js</title>
+<script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript"></script>
+<script type="text/javascript">
+
+  function onOpenCvReady() {
+    document.getElementById('status').innerHTML = 'OpenCV.js is load.';
+    cv["onRuntimeInitialized"]=()=> {
+       document.getElementById("run").disabled=false;
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Please add whitespaces:
```html
document.getElementById("run").disabled = false;
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Please align this bracket

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  }
+  function runBlur(){
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Please add a single whitespace before "{" (codestyle):
```html
function runBlur() {
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    let src = cv.imread('imageSrc');
+    let dst = new cv.Mat();
+    let ksize = new cv.Size(3, 3);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Please rename this variable to `kernelSize` (simpler to understand the meaning)

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 1 above)
Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: done

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 2 above)
Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: done

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 3 above)
Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: done

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 4 above)
Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: done

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 1 above)
Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Really? :)

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  function onOpenCvReady() {
+    document.getElementById('status').innerHTML = 'OpenCV.js is load.';
+    cv["onRuntimeInitialized"]=()=> {
+       document.getElementById("run").disabled=false;
+    }
+  }
+  function runBlur() {
+    let src = cv.imread('imageSrc');
+    let dst = new cv.Mat();
+    let kernelSize = new cv.Size(3, 3);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Is 3x3 kernel enough to see blurring result? I mean shall we use larger area or not?

**Comment by dkurt** [comment]
I think this sample is ready to be wrapped into Markdown.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html (hunk 5 above)
Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: Yes, we can see a result. And yes, we can use lager area.

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html]: @SimpleVlad, the question was mainly about observation. Can user understand that image is blurred and it's enough to use 3x3 kernel?

**Review by dkurt — APPROVED** [review]
:+1: Let's merge it!

**Pull request merged**